Bluepill-cli/Bluepill-cli/Simulator/SimulatorHelper.m

```diff
@@ -55,12 +57,15 @@ + (NSString *)testEnvironmentWithConfiguration:(BPConfiguration *)config {
         BPXCTestFile *xctTestFile = [BPXCTestFile BPXCTestFileFromExecutable:executable withError:&error];
         if (!xctTestFile) {
             [BPUtils printInfo:ERROR withString:[NSString stringWithFormat:@"Failed to load testcases from %@", [error localizedDescription]]];
-            [BPUtils printInfo:WARNING withString:@"Will Run all TestCases"];
+            exit(1);
```

Comment:
exit(1) doesn't match the `BPExitStatus`, 1 means `BPExitStatusTestsFailed`. If we are not expecting a nil xctTestFile here, let's `assert not nil` on that